pepper/Makefile

```diff
@@ -51,7 +52,7 @@ e2e-tests: check-env
 unit-tests: $(patsubst test/apps/%.cpp, app_%, $(wildcard test/apps/*_test.cpp))
 
 app_%_test: test/apps/%_test.cpp
-	$(CXX) $(CXXFLAGS) -DTEST $(TEST_IFLAGS) $< -o $(TEST_BIN)/$@ $(TEST_LDFLAGS)
+	$(CXX) $(CXXFLAGS) -DTEST $(TEST_IFLAGS) $(IFLAGS) $< -o $(TEST_BIN)/$@ $(TEST_LDFLAGS) $(LDFLAGS)
```

Comment:
ditto

Reply:
If, I do not link it, it gives me this error:
test/apps/declarations.h:2:63: fatal error: libsnark/common/default_types/r1cs_ppzksnark_pp.hpp: No such file or directory
 #include <libsnark/common/default_types/r1cs_ppzksnark_pp.hpp>

Reply:
Ah right, because you are now depending on libff in the unit tests as well. Makes sense.